src/utils.py

```diff
@@ -103,4 +107,4 @@ def return_random_number() -> int:
     float
     '''
 
-    return np.random.randint(0, 100)
\ No newline at end of file
+    return np.random.randint(0, 100)
```

Comment:
The new line is an exact copy of the previous line. So, this change doesn't make any sense.